tasks/jsinspect.js

```diff
@@ -92,13 +97,17 @@ module.exports = function(grunt) {
       writableStream.on('finish', function() {
         done(taskSucceeded);
       });
+      writableStream.on('error', function() {
+        throw new Error('Can not write to ' + options.outputPath);
```

Comment:
Ja agree - but we could still pass the original error msg with the custom one, to provide a better experience. :)